lib/include/hse_ikvdb/blk_list.h

```diff
@@ -8,23 +8,14 @@
 
 #include <inttypes.h>
 
-/**
- * struct kvs_block - information about a mblock in a blk_list
- * @bk_blkid:  mblock id
- * @bk_handle: mblock handle
- */
-struct kvs_block {
-    uint64_t bk_blkid;
-};
-
 struct blk_list {
-    struct kvs_block *blks;
+    uint64_t *blks;
```

Comment:
This summarizes the changes in this PR:  I replaced `struct kvs_block` with `uint64_t`.   In most cases there was an array of `struct kvs_block` which was converted to an array of `uint64_t`.  It is mostly a syntax change.  In some cases there is logical change I will point out.

Reply:
some form of this comment is worth putting in the final commit message :)